modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/placeholder_helper.dm

```diff
@@ -1,4 +1,4 @@
 /datum/species
-	var/placeholder_description = "Placeholder Description! Please fill me out!"
-	var/placeholder_lore = "Placeholder Lore! Please fill me out!"
+	var/placeholder_description = "Placeholder Description! Please fill me out! Ask a developer for guidance."
+	var/placeholder_lore = "Placeholder Lore! Come fill me out! Ask a developer for guidance."
```

Comment:
We just gotta remember that lore also has to be verified before being accepted in the game.